### ISSUE TITLE
Tooltip implementation

### ISSUE THREAD

**Issue by scottkleinman** [WheatonCS/Lexos#279]
In the Bootstrap version of Lexos, I have tried to de-commission earlier tooltip methods and switch to Bootstrap tooltips. That works pretty well, but not for the SVG elements in the various visualisations. I have code which successfully combines Bootstrap tooltips (and thus also popovers) with d3.js, but they are a little clunky. The reason is that Bootstrap tooltips cannot track the cursor position. The current version of Lexos uses qTip2 tooltips, which can, but this library refuses to delete tooltips when the cursor leaves the trigger element (try this out, and you'll see). I haven't been able to solve this problem. Another alternative is the d3.tip library. It is supposedly the preferred version of for use with d3.js, but for some reason I was never able to get it to work right last year. There are a couple of other possible solutions.

In summary, Bootstrap tooltips are probably the way to go for the main part of the UI. We should finish the process of implementing them on every screen. However, they are problematic for the d3.js graphs, so we need to come up with some solution for that particular context.

**Comment by scottkleinman** [comment]
This has been implemented.
